packages/vendor-bundle/README.md

```diff
@@ -0,0 +1,60 @@
+# Webpack blocks - vendor bundle
+
+[![JavaScript Style Guide](https://img.shields.io/badge/code%20style-standard-brightgreen.svg)](http://standardjs.com/)
+[![NPM Version](https://img.shields.io/npm/v/@webpack-blocks/vendor-bundle.svg)](https://www.npmjs.com/package/@webpack-blocks/vendor-bundle)
+
+This block for [webpack-blocks](https://github.com/andywer/webpack-blocks) extracts a vendor bundle that includes the most used dependencies of your app.
+
+## Usage
+
+```js
+const { createConfig } = require('@webpack-blocks/webpack')
+const vendorBundle = require('@webpack-blocks/vendorBundle')
+
+module.exports = createConfig([
+  vendorBundle(/* options */)
+])
+```
+
+## Options
+
+#### name *(default: main)*
+
+Name of the main bundle of your app.
+
+#### filename *(default: vendor.[chunkhash].js)*
+
+Output bundle file name.
+
+#### minChunks *(default: 5)*
+
+Minimal number of chunks to include dependency into the bundle.
```

Comment:
At least I didn't get that it means "number of chunks using the dependency" and I am maintainer 😜